std/algorithm/mutation.d

```diff
@@ -1788,15 +1842,30 @@ Params:
 
 Returns:
     a range containing all of the elements of range with offset removed
+
+Note:
+    `remove` only accepts integral types or pairs of integral types as individual
+    offset. Use of other `Offset` types has been $(B deprecated).
  */
 Range remove
 (SwapStrategy s = SwapStrategy.stable, Range, Offset...)
 (Range range, Offset offset)
-if (s != SwapStrategy.stable
-    && isBidirectionalRange!Range
+if (isBidirectionalRange!Range
     && hasLvalueElements!Range
     && hasLength!Range
-    && Offset.length >= 1)
+    && Offset.length >= 1
+    && allSatisfy!(templateOr!(isIntegral, isIntegralPair),  Offset))
```

Comment:
No private symbols in a public constraint - how would the user ever figure what `isIntegralPair` does?